development.md

```diff
@@ -1,10 +1,18 @@
 ---
-title: Development information
+title: Developer information
 ---
 
-# Development information
+# Developer information
 
-To build your own Delta Chat bot is easy. Read on for some basic information and maybe have a look at our example bots. You may also copy and adapt them, if that suits you! They're mostly licensed under the GPL, which allows you to use the code, and also to change it as long as you re-publish your result (at least on request).
+Building your own Delta Chat bot is easy.
+Read on for some basic information and maybe have a look at our example bots.
+You may also copy and adapt them, if that suits you!
+
+They're mostly licensed under the GPL, which allows you to use the code, and also to change it as long as you re-publish your result (at least on request).
+
+A good starting point is this [Hello World Example](https://py.delta.chat/examples.html#playing-around-on-the-commandline).
```

Comment:
Its **not** a good example for bots, because the tricky part is missing: receiving and responding to messages.

Reply:
hello world examples are not about tricky parts :)

That would fit very well into the TODO item at the bottom of development.md, in the event-answer-roundtrip. Can you write something about that?

Reply:
> hello world examples are not about tricky parts :)

they should get you running for experimentation, if half the logic that is required for a bot is missing it is not fun.

> Can you write something about that?

not at the moment, also that doesn't really solve the problem of the hello world example being a bad example that does demonstrate any bot-like behaviour.